bin/buildBlueprints.js

```diff
@@ -12,37 +13,36 @@ var makeBuild = require('../lib/makeBuild').makeBuild;
 var configs = require('../lib/configs');
 var getWebpackEntryForTest = require('../lib/getWebpackEntryForTest');
 
+var TEST_DIR = './.test';
+var PRODUCTION_ENV = 'production';
+var TARGETS = {
+  TEST: 'test',
+  CLIENT: 'client',
+  SERVER: 'server',
+};
+
 var argv = require('yargs')
   .alias('b', 'blueprintsPath')
     .describe('b', 'path to a raw-config via a node file with moduel.exports = config')
     .default('b', './blueprints.config.js')
-  .alias('p', 'production')
-    .describe('p', 'enable production settings for the default build cofings')
-    .default('p', false)
-  .alias('c', 'client')
-    .describe('c', 'use the default client build, assumes you have an entry point to a client at ~/lib/client.[some es6.js or .js or .jsx]')
-    .default('c', false)
-  .alias('s', 'server')
-    .describe('s', 'use the default server build, assumes you have an entry point to a server at ~/lib/server[some es6.js or .js or .jsx]')
-    .default('s', false)
-  .alias('a', 'clientAndServer')
-    .describe('a', '[DEFAULT=true] use both a client and a server build. checks if you have an extend build and applies it.')
-    .default('a', true)
   .alias('w', 'watch')
     .describe('w', '[DEFAULT=false] force watching of all builds')
     .default('w', false)
   .alias('i', 'ignoreBlueprints')
     .describe('ignore the blueprints.config.js file in the current directory and use defaults')
     .default('i', false)
-  .alias('t', 'runTest')
```

Comment:
The existing options allowed for passing values that should be mutually exclusive.  You couldn't have it run the tests as well as build for production afterwards for example.